core/Enclave/key_operation.cpp

```diff
@@ -837,18 +893,20 @@ sgx_status_t ehsm_rsa_decrypt(const ehsm_keyblob_t *cmk,
         ret = SGX_ERROR_UNEXPECTED;
         goto out;
     }
-
+log_i("whh");
```

Comment:
remove

Reply:
yes, other comment like this remove also